### DISCUSSION TITLE
What does is requeueing mean, when executing the datastore import?, and why does this occur?

### DISCUSSION THREAD

**Discussion by tridoxx** [GetDKAN/dkan#3873] [Q&A]
hello, I am doing dataset imports, with the command drush queue:run datastore_import
, and I see that some datasets show this, I would like to know why this happens. Thanks a lot

![image](https://user-images.githubusercontent.com/22879539/200980221-c2ef9029-43a0-4a23-821f-f3d7a702afc3.png)

**Comment by janette** [comment]
@tridoxx the default importer is parsing the data into a datastore table row by row, and if the file is large it may exceed the batch limit, once the limit is hit the remaining rows are re-queued and wait for the next cron run to resume.

If you are using a mysql database and your db user has permission to use LOAD DATA LOCAL INFILE, then you can enable the [datastore_mysql_import module](https://github.com/GetDKAN/dkan/tree/2.x/modules/datastore/modules/datastore_mysql_import), this will import the file in a single step and is MUCH faster than the default option.

**Comment by tridoxx** [comment]
@janette Thank you very much for your answer, so just by enabling the module and that the user has the permission, I execute the harvest normally, and it should make the import much faster?

**Reply by tridoxx** [reply]
well yes, I activated the module, then it gives the user permissions in the database, and it goes much faster, thank you very much for the help @janette
